scripts/build_xsvm.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! [[ "$0" =~ scripts/build_xsvm.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+source ./scripts/constants.sh
+
+PLUGIN_FILENAME="v3m4wPxaHpvGr8qfMeyK6PRW3idZrPHmYcMTt7oXdK47yurVH"
+
+# Set the PATHS
+GOPATH="$(go env GOPATH)"
+if [[ $# -eq 1 ]]; then
+    BINARY_DIRECTORY=$1
+elif [[ $# -eq 0 ]]; then
+    BINARY_DIRECTORY="${GOPATH}/src/github.com/ava-labs/avalanchego/build/plugins/${PLUGIN_FILENAME}"
+else
```

Comment:
Can we put this in the default plugin location? `~/.avalanchego/plugins`. Putting it into the avalanchego build directory is a remnant of the (very) old default plugin location.